src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Button.css';
+
+function Button({type, disabled, action, text}) {
+    return (
+        <button
+            type={type}
+            disabled={disabled}
+            onClick={action}
+        >
+            {text}
+        </button>
+    );
+}
```

Comment:
Dit voelt voor mij als een doorgeefluik. Het bespaart geen code, maar in tegenstelling heb je nu méér code